main/get.c

```diff
@@ -829,7 +830,8 @@ extern int cppGetc (void)
 				int next = getcFromInputFile ();
 
 				if (next == NEWLINE)
-					continue;
+					// continue; <-- this may exit returning the backslash, which si wrong.
```

Comment:
It will take your time but I write what I want here.
I guess your change about get.c is for fixing a generic bug.
I would like to you make a separate pull request fixing only the bug. The pull request should
include a test case verifying the fix.
Making a separate pull request takes time. So you may want keep the change in your current working branch. Sometimes I have done so. However, I always make a separate commit when fixing a generic bug in such case. Here I will call the former one "separate-pr method", and the later "separate-commit method". If you don't know I would like you to know "separate-pr method" first.

separate-pr method:

1. checkout master
2. pull upstream (universal-ctags/ctags)
3. make a branch for fixing the generic bug only and checkout the branch
4. fix the bug, and make a commit
5. add a test case, and make a commit
6. push it to your github repo
7. make a PR
8. merge the PR to the upstream. I will do this time, however, you may do in the future.
9. checkout master
10. pull upstream (your commit for fixing the bug will be merged to your local repo)
11 checkout cxx-parser branch
12 rebase the cxx-parser branch on your master
13. you will got conflicts. Solve them.
14. git push --force origin cxx-parser:cxx-parser
15. let me know the pushing

I'm not sure these steps are the best or not. However, I have done so while working on ctags.
If you don't know I recommend you to learn "git rebase -i/--continue/--abort", "git push --force",
and "git add -p".

separate-commit method:
1. git rebase -i master, and mark "edit" on the commi where you introduced the fix for the bug.
2. git add main/get.c
3. write a test case for the fix
4. Do git add the test case
5. make a commit for the fix
6. look at the output of git status
7. do git add the rest of changes
8. git rebase --continue

Good luck!